src/wktplot/wktplot.py

```diff
@@ -23,13 +23,14 @@
 class WKTPlot:
     logger = logging.getLogger(__name__)
 
-    def __init__(self, title: str, save_dir: Union[str, Path]):
+    def __init__(self, title: str, save_dir: Union[str, Path], save_fig: bool =True):
```

Comment:
It doesn't quite make sense to have both `save_dir` and `save_fig` arguments. If `save_fig` is `False`, then the value for `save_dir` still has to be a valid directory even if it's not being used.

Reply:
I think for implementing this feature request, I would support making the `save_dir` argument nullable, and set `output_file` if the value is not null and a valid directory.